### ISSUE TITLE
[Bug]: Can't download JASP

### ISSUE THREAD

**Issue by Ferg72** [jasp-stats/jasp-issues#1737]
### JASP Version

0.16.2

### What OS are you seeing the problem on?

Mac OS Monterey V12.4

### Bug Description

I've tried to download the latest version of JASP, and once it is visible in the bar, the app will not open, and a 'the app is not responding' message is shown (see screenshot).

I've tried downloading an older version (16.0), and using a different browser (Safari and Chrome), but same issue with both.

<img width="284" alt="Screenshot 2022-06-04 at 9 30 31 pm" src="https://user-images.githubusercontent.com/106857680/171997200-09136096-e2e0-4a89-9ff7-5c3befc52581.png">

### Expected Behaviour

The app should just open when I click on it - no issue with the older version that I had (14.3, I think).

### Steps to Reproduce

1. Go to website and click on download (Mac0S, Apple Silicon)
2. Double click dmg file
3. Move JASP to application folder
4. Open application folder and double-click to open > bug as above, rather than opening.

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by Ferg72** [comment]
Note, having gone through old posts, I typed a path into terminal as suggested, and this is the output, if that helps:
cd /Application/JASP.app/Contents/MacOS./JASP
cd: no such file or directory: /Application/JASP.app/Contents/MacOS./JASP

**Comment by tomtomme** [comment]
I do not use Mac, but some of my students were successful with this guide: **https://jasp-stats.org/installation-guide/**
Or did you try that already?
Also: Which version of 0.16.2 did you download? This could help the devs figure out, whats wrong...

**Comment by Ferg72** [comment]
Thanks tomtomme - I tried that guide, but no success.
I have been downloading the MacOS Intel (10.5+) version, as my Macbook runs a 3.1 GHz Quad-Core Intel Core i7 processor.

**Comment by Ferg72** [comment]
Thanks @boutinb.

You're correct that there was initially confusion about the version, but realising that, I removed it from the applications folder, and downloaded the Intel MacOS version, and the same thing happens - i.e. the JASP icon appears in the dock, bounces for a while and then stops.  The app doesn't open, and when I right click the icon, "Application not responding" appears in greyed out text at the top of the window.

When I type this into terminal:

cd/Application/JASP.app/Contents/MacOS./JASP

The output is: 

zsh: no such file or directory: cd/Application/JASP.app/Contents/MacOS./JASP
ianferguson@MacBook-Pro ~ %

As you say, I think this means that I'm unable to install JASP, and I just can't seem to troubleshoot this issue.

Ian

**Comment by boutinb** [comment]
Hi @Ferg72, let's try to do the things step by step, and tell me for each step what happens.

1.  Do you still see in the dock bar the JASP icon. If yes, you should first right-click the icon, and choose the Quit or Force Quit menu option. It must disappear first.
2. Afterwards, with the Finder, if you go to Applications, do you see the JASP icon? If you see it, remove it.
3. Then, download (once again) the Intel Mac OS version of JASP.
4. When you open the dmg file, you should see maybe a warning that this file comes from internet, but you should be able to choose 'Open'
5. Then you should see this: 
![Screenshot 2022-06-08 at 10 39 33](https://user-images.githubusercontent.com/503332/172572268-5ea84015-a0ad-4972-8e66-0e333e9113ca.png)
7. Then you drag & drop the JASP icon to the Applications icon.
8. After JASP is copied to the Applications folder, you should be able to double-click the JASP icon in the Applications folder.
9. If JASP cannot start, then with a terminal, type this: `/Applications/JASP.app/Contents/MacOS/JASP` (no cd before): does JASP starts? If not, could you give me the output of the terminal.

**Comment by Ferg72** [comment]
Hi @boutinb,

In terms of your steps:

1. Done - the icon is in the bar, but removed after force quit.
2. The JASP icon is still in Applications at this point, but removed.
3. Downloaded without issues
4. File opens, without any warnings.
5. The picture appears.
6. No issues dragging into Applications folder
7. No issues with double-clicking the JASP icon.
8. JASP doesn't start - back to the bouncing icon, with an eventual "application not responding" when this stops.
The terminal doesn't give any additional output (and I've left it for a long time), and the following message comes up when attempting to close the terminal
<img width="584" alt="Screenshot 2022-06-09 at 12 41 52 pm" src="https://user-images.githubusercontent.com/106857680/172753770-e7fadf52-96ae-4be3-8b9f-5e1c94d473c1.png">
<img width="570" alt="Screenshot 2022-06-09 at 12 41 44 pm" src="https://user-images.githubusercontent.com/106857680/172753787-e3af6ab5-bcd2-4f97-858d-672d89739bc3.png">
.

**Comment by Ferg72** [comment]
I'm getting the feeling this may be an intrinsic issue with my Mac, rather than a problem with me or the App.

Thanks for your help.

Ian

**Comment by boutinb** [comment]
This is quite weird indeed....
Just to be sure: in the Applications folder, do you have only 1 JASP application?
What I suppose is going wrong on your Mac is when the Gatekeeper tries to check whether JASP is verified (signed and notarized). For this it needs an internet connection. Do you have special settings on your Mac, is it your personal Mac, or a Mac given (and formatted) by your company?

**Comment by Ferg72** [comment]
Yep, very weird, but I've checked and same thing with R Studio, so I'm pretty sure it's my computer.

There is only one JASP application in my folder.

It's a personal mac, and I've had JASP before - this was, frustratingly just an update...

Thanks anyway for your help.

**Comment by tomtomme** [comment]
@Ferg72 
Could you troubleshoot the issue in the meantime? Did you try a newer JASP version or an MacOS update? Current JASP 0.18.3 supports MacOS 12 to 14
